conftest.py

```diff
@@ -93,6 +99,12 @@ def pytest_configure(config):
     )
 
     project_id = config.getoption("--gcp-project-id")
+    folder_id = config.getoption("--gcp-folder-id")
+    if project_id is not None and folder_id is not None:
+        # TODO: Probably a better way to cleanly exit than `exit(1)`?
```

Comment:
Maybe `raise Exception("Invalid arguments. Cannot use --gcp-fol.. and --gcp-proj..")`